### DISCUSSION TITLE
Disable contents of whole form

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10615] [CLI - PWA mode]
Hello Friends ,

I have a component that contain many input fields and dropdowns , I want to make the whole component read-only.
Is there a faster way to disable those components without adding the disable key word in each input's and dropdown's ?

Can someone please help 

Thanks

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 

> Is there a faster way to disable those components

give all your inputs a ref , put all the refs in an array, for each ref in the array,  set the input on disabled.

**Reply by akashramtohul4** [reply]
I dont know if its possible to do that by css

**Reply by akashramtohul4** [reply]
i want to disable entire form

**Reply by MilosPaunovic** [reply]
Who said anythinfg about CSS? @ontwikkelfabriek suggested `refs`.

**Reply by akashramtohul4** [reply]
I found a better solution. The whole component looks like disabled  

https://codepen.io/metalsadman/pen/QRoKoz
